testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java

```diff
@@ -0,0 +1,93 @@
+package org.jboss.resteasy.test.providers.html;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.test.providers.html.resource.HeadersInViewResponseResource;
+import org.jboss.resteasy.utils.PortProviderUtil;
+import org.jboss.resteasy.utils.TestUtil;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @tpSubChapter HTML provider
+ * @tpChapter Integration tests
+ * @tpSince RESTEasy 3.1.3.Final
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class HeadersInViewResponseTest {
+
+    static ResteasyClient client;
+
+    @Deployment()
+    public static Archive<?> deploy() {
+        WebArchive war = TestUtil.prepareArchive(HeadersInViewResponseTest.class.getSimpleName());
+        war.addAsLibrary(getResteasyHtmlJar());
```

Comment:
Well, I can't think of a compelling reason one way or the other. MavenUtil is way more complicated and, presumably, more general. You could argue that using the maven repository is immune to Resteasy restructuring, but how often are we going to move resteasy-html?